### DISCUSSION TITLE
Ability to update profile status via GitHub API

### DISCUSSION THREAD

**Discussion by mack** [community/community#15883] [Other Feature Feedback, Questions, & Ideas] [answered]
Currently, the [GitHub API](https://docs.github.com/en/rest/users/users#update-the-authenticated-user) does not appear to allow you to programmatically update your profile status. I think adding this functionality could be really useful for a lot of people who have 3rd party GitHub apps, especially since it's the most frequently updated piece of information on a profile.

**Comment by Valkyrie-Akness** [comment, marked as answer]
Yes, an update was once possible, but GitHub decided to cut this function from the API. It may be worth contacting support and suggesting the return of this function?
